crates/agglayer-storage/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+// Physical storage
+#[rustfmt::skip]
+pub mod storage;
```

Comment:
Does the `rustfmt::skip` attribute apply only here, or to the whole module, including the contents in `storage.rs`?

Reply:
This is temporary to not fail the CI with empty files, until the implementation PR are set